<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerPage</class>
 <widget class="QDialog" name="CustomerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_panamera">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Panamera </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_macan">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>80</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Macan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_checkou">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Checkout</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_911">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>911</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_718">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>718</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cayenne</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_availability">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Check availability</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
